autoware_sensing_msgs/msg/RadarFieldInfo.msg

```diff
@@ -0,0 +1,8 @@
+std_msgs/String field_name
+bool min_value_available
```

Comment:
For the availability bools, I remember there was a discussion about using NaN (I guess the above suggestion by @xmfcx , using `-1.0f` may work for resolution, but it's a bit dangerous as requires the user to check an apparently irrelevant field for availability). I'm guessing having the bools as proposed is cleaner?

Reply:
Another way of doing this would be to use a field like
`float32[<=1] min_value`
which is functionally equivalent to an `std::optional<float32>`.
This would remove the need for the boolean field, and the user is forced to check if the value is present. Probably a bit ugly to use since it shows up as a dynamic bounded **array**, not an optional :smiling_face_with_tear: